gwlfe/gwlfe.py

```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+import logging
+
+
+log = logging.getLogger(__name__)
+log.setLevel(logging.DEBUG)
+
+
+def repeat_schema(times, *cols):
+    # `times` may be an int or a function which returns an int (the value of
+    # which may not be available until we start parsing the GMS file).
+    t = lambda result: times(result) if callable(times) else times
+
+    def repeat_schema_inner(result):
+        times = t(result)
+        for _ in range(times):
+            for col in cols:
+                yield col
+    return repeat_schema_inner
+
+
+def get_value(name):
+    def get_value_inner(result):
+        return result[name]
+    return get_value_inner
+
+
+EOL = '<eol>'
+
+
+NO = 0
+YES = 1
+
+
+class ET_FLAG(object):
+    HAMMON_METHOD = 0
+    BLAINY_CRIDDLE_METHOD = 1
+
+
+class GROW_FLAG(object):
+    NON_GROWING_SEASON = 0
+    GROWING_SEASON = 1
+
+
+class LANDUSE(object):
```

Comment:
These land use values are specified by strings in the GMS file but there will likely be a transformation phase where we change this data to a more suitable format.